source/collections/time-series-collection.txt

```diff
@@ -0,0 +1,88 @@
+.. _time-series-collection:
+
+===============================
+Create a Time Series Collection
+===============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+:manual:`Time series collections </core/timeseries-collections/>` efficiently 
+store sequences of measurements over a period of time. 
+
+Procedure
+---------
+
+.. procedure::
+   :style: connected
+
+   .. step:: Click the :guilabel:`Create Collection` button.
+
+      From the :guilabel:`Collections` screen, click the 
+      :guilabel:`Create Collection` button.
+
+   .. step:: Enter the collection name.
+
+   .. step:: Click the :guilabel:`Advanced Collection Options` dropdown.
+
+      Check the :guilabel:`Time Series Collection` option.
+
+   .. step:: Specify a :guilabel:`timeField`.
+
+      Specify which field should be used as the ``timeField`` for the time-series 
+      collection. This field must have a :manual:`BSON type date </reference/bson-types/>`.
+
+      The following fields are optional:
+
+      .. list-table::
+         :header-rows: 1
+         :widths: 50 50 
+
+         * - Field
+           - Description 
+
+         * - ``metaField``
+           - The name of the field which contains metadata in each time series 
+             document. The metadata in the specified field should be data that 
+             is used to label a unique series of documents. 
+
+         * - ``granularity``
+           - The ``granularity`` field allows specifying a coarser granularity so 
+             measurements over a longer time span can be more efficiently stored 
+             and queried.
+
+             The default value is set to ``seconds``.
+
+         * - ``expireAfterSeconds``
+           -  The ``expireAfterSeconds`` field enables automatic deletion of 
+              documents older than the specified number of seconds.  
+
+      For more information on time series fields, see 
+      :manual:`Time Series Collections </core/timeseries-collections/>`.
+
+   .. step:: Click :guilabel:`Create Collection` to create the collection.
+
+      Your collection will be marked by a :guilabel:`time series` badge. On the 
+      left-hand navigation bar your time series collection will be have the 
+      following icon: 
+
+      .. figure:: /images/compass/time-series-badge.png
+         :figwidth: 450 px
+         :alt: Time Series Icon
+
+Restrictions and Limitations
+----------------------------
+
+The following restrictions and limitations apply when creating a time series 
+collection:
+
+- You can only use :guilabel:`custom collation` along side your capped 
```

Comment:
[minor typo]

I think this ought to be "alongside" on line 84.

Reply:
Done.